lib/twilio-ruby.rb

```diff
@@ -0,0 +1,12 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'Time'
+
+account_sid = ENV['TWILIO_ACCOUNT_SID']
+auth_token = ENV['TWILIO_AUTH_TOKEN']
+@client = Twilio::REST::Client.new account_sid, auth_token
+time = Time.now
+@client.account.messages.create(
+body: "Thank you!  Your order was placed and will be delivered before #{(time.hour) + 1}:#{time.min}",
```

Comment:
Line is too long. [102/80]<br>Indent the first parameter one step more than the previous line.